cmd/nomos/hydrate/hydrate.go

```diff
@@ -105,6 +106,9 @@ which you could kubectl apply -fR to the cluster, or have Config Sync sync to th
 
 		if sourceFormat == filesystem.SourceFormatHierarchy {
 			files = filesystem.FilterHierarchyFiles(rootDir, files)
+		} else {
+			// hydrate as a root repository to preview all the hydrated configs
+			options.Scope = declared.RootReconciler
```

Comment:
Why is the scope related to source format?

Reply:
The scope is used by namespace selectors and other validators for unstructured source format.

The hierarchy mode doesn't rely on the scope because the folder structure has already defined the scope.

Reply:
Before the change, it uses a `IsNamespaceReconciler` flag, which is semantically the same as scope.
It wasn't set explicitly here because the flag is default to false, which indicates a root scope.